modules/features/account/src/main/java/au/com/shiftyjelly/pocketcasts/account/viewmodel/OnboardingUpgradeFeaturesViewModel.kt

```diff
@@ -103,14 +103,12 @@ class OnboardingUpgradeFeaturesViewModel @Inject constructor(
 
         val upgradeLayout = when {
             showPatronOnly -> UpgradeLayout.Original
-            FeatureFlag.isEnabled(Feature.PAYWALL_AB_EXPERIMENT) -> UpgradeLayout.Reviews
-            FeatureFlag.isEnabled(Feature.PAYWALL_AA_EXPERIMENT) -> {
-                val variation = experimentProvider.getVariation(Experiment.PaywallAATest)
-                // For the A/A test show the same layout for both variations
+            FeatureFlag.isEnabled(Feature.EXPLAT_EXPERIMENT) -> {
+                val variation = experiments.getVariation(Experiment.PaywallAATest)
                 if (variation == Variation.Control) {
                     UpgradeLayout.Original
                 } else {
-                    UpgradeLayout.Original
+                    UpgradeLayout.Reviews
```

Comment:
I am setting this to `Reviews` for testing purposes, but we might want to change it to `UpgradeLayout.Original` so we can run an AA test for Explat

Reply:
Probably since it's an A/A Explat we want both variation to have the same experience and only test that the assignment process is working properly

Reply:
Yep! I was planning to revert this before merge this PR. It's done now! Thanks for checking this